digits/job.py

```diff
@@ -266,3 +266,10 @@ def is_read_only(self):
         Returns False if this job can be edited
         """
         return False
+
+    def parent_jobs(self):
+        """
+        Returns the list of the jobs on which the job
+        depends
+        """
+        raise NotImplementedError('Implement me!') #@TODO
```

Comment:
I was just following the repository's convention. But i agree the exceptions are quite dirty, will change.
https://github.com/NVIDIA/DIGITS/blob/master/digits/job.py#L152-L157